### ISSUE TITLE
php-pr0xy.herokuapp.com

### ISSUE THREAD

**Issue by moto6333** [portaltree/php-Pr0xy#1]
It seems that php-pr0xy.herokuapp.com is down making it so no one can run this

**Comment by portaltree** [comment]
yes I stopped hosting since it costed money. Just run it by yourself

**Comment by moto6333** [comment]
Thats the thing unless I am being a idiot and not setting it up right it uses svg files hosted on heroku and cant be access self hosted
